DevEdu.API/DevEdu.API/Controllers/TaskController.cs

```diff
@@ -124,6 +126,9 @@ public int UpdateStatusOfStudentAnswer(int taskId, int studentId, int statusId)
         [HttpPost("{taskId}/student/{studentId}/comment")]
         public int AddCommentOnStudentAnswer(int taskId, int studentId, [FromBody] CommentAddInputModel inputModel)

```

Comment:
I think we can simplify this
let assume that we already have id of Task_Student entity somewhere on UI-side
so lets change the route to "answer/{taskStudentId}/comment")